Source/WebGPU/WebGPU/Buffer.h

```diff
@@ -148,6 +152,23 @@ class Buffer : public WGPUBufferImpl, public RefCounted<Buffer>, public CanMakeW
 #if CPU(X86_64)
     bool m_mappedAtCreation { false };
 #endif
+#if ENABLE(WEBGPU_SWIFT)
+} SWIFT_SHARED_REFERENCE(retainBuffer, releaseBuffer);
+#else
```

Comment:
I believe SWIFT_SHARED_REFERENCE is already a macro that compiles to nothing if you're not building with Swift-C++ interop enabled. If it's not, let's make a WebKit macro.

The we can just say "MACRO(retainBuffer, releaseBuffer)", without the ENABLE guard.